src/ansys/fluent/core/launcher/container_launcher.py

```diff
@@ -1,4 +1,16 @@
-"""Provides a module for launching Fluent in container mode."""
+"""Provides a module for launching Fluent in container mode.
+
+Examples
+--------
+
+>>> from ansys.fluent.core.launcher.launcher import create_launcher
+
+>>> container_meshing_launcher = create_launcher("container", mode="meshing")
+>>> container_meshing_session = container_meshing_launcher()
+
+>>> container_solver_launcher = create_launcher("container")
+>>> container_solver_session = container_solver_launcher()
+"""
```

Comment:
`create_launcher()` method was mentioned in `Fluent launcher` but it's examples were missing.

Reply:
![image](https://github.com/ansys/pyfluent/assets/106588300/18d1f043-8a67-4ba4-959f-90f5e53e7e8c)